src/test/java/net/minecraftforge/debug/world/PotentialSpawnsEventTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.debug.world;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.Difficulty;
+import net.minecraft.world.entity.MobCategory;
+import net.minecraft.world.level.Level;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.world.WorldEvent;
+import net.minecraftforge.fml.common.Mod;
+
+@Mod("potentialspawnsevent_test")
+public class PotentialSpawnsEventTest
```

Comment:
Please provide a javadoc for this class which explains what feature this tests, how to test it, and what the expected results are from said test.